### DISCUSSION TITLE
W 2023 r zmieniamy przeglądarkę...

### DISCUSSION THREAD

**Discussion by hawkeye116477** [MajkiIT/polish-ads-filter#20291] [General]
W Styczniu 2023 uBO przestanie działać w przeglądarce wujaszka G. Tak więc, co ci mają tę przeglądarkę będą musieli ją zmienić na jakąś inną, jeżeli alternatywa nadal będzie, bo jak wiadomo Mozilla lubi naśladować Google. No chyba, że inne przeglądarki bazujące na Chromium odwrócą złe zmiany... 

Ewentualnie może chociaż aplikacja AG na Windę nadal będzie działać poprawnie z każdą przeglądarka.

Źródło: https://www.bleepingcomputer.com/news/google/google-manifest-v2-chrome-extensions-to-stop-working-in-2023/

**Comment by xxcriticxx** [comment]
@hawkeye116477 poradzimy sobie

**Reply by hawkeye116477** [reply]
Szczególnie jak będzie Neighborhood Browser autorstwa @xxcriticxx, fork Chromium z revertem zmian wujka G :smile:

**Reply by mjholub** [reply]
A czemu nie degoogled chromium po prostu?

**Reply by xxcriticxx** [reply]
ten projekt?

https://github.com/Eloston/ungoogled-chromium

**Reply by mjholub** [reply]
Tak. Chociaż ja od dawna nie używam, przeglądarki oparte na Chromium/Chrome nigdy do mnie specjalnie nie przemawiały, plus miałam problem z instalowaniem rozszerzeń i zapamiętywaniem ustawienia wyszukiwarki, zwłaszcza po tym jak G**gle wyłączyło możliwość pobierania .crx, nawet za pośrednictwem proxy. Moją główną przeglądarką obecnie jest Firedragon, jak potrzebuję WebGL albo Widevine to z kolei odpalam Brave (znaczy FD ma domyślnie WGL włączony, ja wyłączam żeby zmniejszyć fingerprinting). Dla mnie dużą wadą  przeglądarek opartych na Chromium jest brak container tabs, chociaż zauważyłam że Facebook chyba za pośrednictwem jakichś superciasteczek obchodzi tę funkcję, dlatego zablokowałam go kompletnie za pomocą blocklisty Stevena Black w hosts. 

Martwi mnie co przyniesie zastąpienie Androida Fuchsią, patrząc po ostatnich poczynaniach tego dystopijnego korpo (inb4 masło maślane, kto jak kto ale to Google szczególnie zasługuje na takie miano z ich tendencjami monopolistycznymi, ograniczaniem swobody użytkowników, masowym szpiegowaniem wpychającym się wszędzie jak nie drzwiami to oknem, tak że zablokowanie ich w hosts może nam odciąć dostęp do wielu stron używających ich captchy i paroma innymi przewinieniami). Co prawda spoko że wprowadza mikrokernel i zastępuje Javę C++em dla aplikacji end-user, ale mam obawy co do swobody modyfikacji systemu przez użytkowników. Czy czeka nas zmierzch custom ROMów i pozbycie się GSF już nie będzie takie proste? Owszem, sam system jest oparty na wolnych licencjach, jest open source, ALE czy nie wprowadzi jakichś rozwiązań, które będą sukcesorem zamkniętoźródłowego GSF, którego reverse engineering w celu stworzenia czegoś na wzór microG, tak aby bez tego zamkniętoźródłowego spyware dało się używać połowy aplikacji będzie takie proste? Dobrze że język dla aplikacji to C++, to może nawet mimo tego mikrojądra będzie łatwiej przeportować FOSS apki z linuxa.

**Comment by mjholub** [comment]
Mozilla trochę włazi w d... Google, choćby poprzez wprowadzony Safe Browsing. Mają też parę innych nieprzyjaznych prywatności rozwiązań, jak Pocket, telemetria czy domyślna wyszukiwarka ustawiona właśnie na Google. Ale wprowadzają też trochę pozytywnych rozwiązań, jak chociażby Container Tabs, czy "zaawansowana" ochrona przed śledzeniem. Jako pierwsi chyba wprowadzili też DoH i powszechny HTTPS. [Privacy Guides](https://privacyguides.org/) trochę o tym pisze, nieco jednak trochę wybielając mozillę, biorąc czytelnika za debila, którego threat model powinien brać pod uwagę że łatwo może dać się zphishingować etc. i że to cena, którą warto zapłacić (tak jakby się nie dało zasubskrybować blocklist stron phishingowych 😏). 
Całe szczęście mamy GNU IceCat, Librewolf czy Firedragon, a do tego [Firefox Profilemaker](https://ffprofile.com/). Zresztą wracając – nie wydaje mi się, by Mozilla prędko zdecydowała się na taki krok, kiedy uBO ma obecnie status polecanego rozszerzenia na ich stronie z rozszerzeniami. Nie porównywałabym fundacji, nawet mającej co nieco za uszami, do gargantuicznego korpo, którego lwią część przychodów stanowią właśnie reklamy.

**Reply by xxcriticxx** [reply]
@154pinkchairs to była bardzo długa odpowiedź

**Comment by hawkeye116477** [comment]
Kolejny powód, by się odgooglować.

https://www.steamgifts.com/discussion/d2DuB/hopefully-sg-wont-follow-this-trend
